### ISSUE TITLE
Mega Menus don't work in Sidebar Menu Mode (Astroid template two)

### ISSUE THREAD

**Issue by rkrisztian** [templaza/astroid-framework#814]
Mega Menus don't work when the Header Mode is set to Sidebar Menu Mode, at least when using the Astroid Template two. There is no on-hover effect that pops out the submenu items, just a plus sign that I don't even consider acceptable. This is a huge setback for those who plan to have many menu entries and prefer the menu on the side.

 - Joomla Version: 5.1.4
 - PHP Version: 8.1
 - Astroid Version: 3.1.1, with the Framework extension updated via Joomla extension manager. (There were no 3.1.3 equivalent of 
`astroid-package-v3.1.1.Framework-Template.zip` to install.)
 - Device: PC
 - OS: Linux
 - Browser: LibreWolf

**Comment by sonvnn** [comment]
Yes, because Sidebar layout does not support Pops out submenu Items. It's original from Astroid 2. Thank you for your feedback. I will add this to tasklist for the future update :)
